edb/schema/types.py

```diff
@@ -2910,6 +2910,16 @@ def ensure_schema_type_expr_type(
     return cmd
 
 
+def type_dummy_expr(
+    typ: Type,
+    schema: s_schema.Schema,
+) -> Optional[s_expr.Expression]:
+    if isinstance(typ, so.DerivableInheritingObject):
+        typ = typ.get_nearest_non_derived_parent(schema)
+    text = f'assert_exists(<{typ.get_displayname(schema)}>{{}})'
+    return s_expr.Expression(text=text)
```

Comment:
Woah, we can construct schema objects directly using the constructor? Didn't know that.

Reply:
`Expression` isn't really a schema object, I think